src/v2/components/FireListPagination.jsx

```diff
@@ -0,0 +1,56 @@
+// -----------------------------------------------------------------------------
+// Copyright 2022 Open Climate Tech Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+import React, {useCallback} from 'react'
+
+import Button from './Button.jsx'
+import ButtonGroup from './ButtonGroup.jsx'
+import ButtonOutput from './ButtonOutput.jsx'
+
+/**
+ * Provides a pagination button group for flipping back and forth between each
+ * fire in a list.
+ *
+ * @param {Object} props
+ * @param {Array} fires - The list of fires to page through.
+ * @param {function(number)} onScrollToFire - Event handler to be called when
+ *     when a fire at a specific index should scroll itself into view.
+ * @param {number} props.selectedIndex - The index of the active fire.
+ *
+ * @returns {React.Element}
+ */
+export default function FireListPagination(props) {
+  const {fires: {length: nFires}, onScrollToFire, selectedIndex} = props
+
+  const handleNext = useCallback(() => {
+    onScrollToFire(selectedIndex + 1)
+  }, [selectedIndex, onScrollToFire])
+
+  const handlePrev = useCallback(() => {
+    onScrollToFire(selectedIndex - 1)
+  }, [selectedIndex, onScrollToFire])
+
+  const pagination = nFires > 0 ? `Fire ${selectedIndex + 1} of ${nFires}` : '…'
```

Comment:
nFires is going to age as new fires are detected and old fires age out.  Maybe we should just have prev/next buttons without nFires being displayed?

Reply:
I think it’s important for these sorts of controls to have a count because it gives users both a context and a landmark to orient themselves. So I’d like to take up the fire aging case as a UX design problem. When new fires come in, we’ve already agreed that we wait for the user to acknowledge a banner before we update the list—in that case, in addition to adding new fires, I assume we will remove old ones. I haven’t thought it through, but a similar albeit less intrusive acknowledgement might work for aging out fires.

Reply:
Added TODO